### DISCUSSION TITLE
How to get Parent tag name and I'd?

### DISCUSSION THREAD

**Discussion by san-123** [GrapesJS/grapesjs#3920] [General]
Hi @artf ,

            I have content below,

```
`<Div id="Maindiv">
<Ul I'd="ULid>
<Li>test elements</li>
</Ul>
</Div>`
```

When I select ul then I need to get Parent div id and tag name. Please how to achieve this.

**Comment by artf** [comment]
`component.parent().get('tagName')/getId()`
